internal/rpc/rpcserver/server.go

```diff
@@ -3352,6 +3353,128 @@ func (s *votingServer) SetVoteChoices(ctx context.Context, req *pb.SetVoteChoice
 	return resp, nil
 }
 
+// treasuryPolicies returns voting policies for treasury spends for all keys in an array
+func (s *votingServer) TreasuryPolicies(ctx context.Context, req *pb.TreasuryPoliciesRequest) (*pb.TreasuryPoliciesResponse, error) {
+	policies := s.wallet.TreasuryKeyPolicies()
+	resp := &pb.TreasuryPoliciesResponse{
+		Policies: make([]*pb.TreasuryPoliciesResponse_Policy, len(policies)),
+	}
+	for i := range policies {
+		var policy string
+		switch policies[i].Policy {
+		case stake.TreasuryVoteYes:
+			policy = "yes"
+		case stake.TreasuryVoteNo:
+			policy = "no"
+		}
+		r := &pb.TreasuryPoliciesResponse_Policy{
+			Key:    hex.EncodeToString(policies[i].PiKey),
+			Policy: policy,
+		}
+		resp.Policies = append(resp.Policies, r)
```

Comment:
either this append needs to change or the creation of the slice above needs to, because currently this is appending to a bunch of zero results and will result in twice as many policies in the response, with the first half all zeros